### ISSUE TITLE
Problems with parsing

### ISSUE THREAD

**Issue by popiolek** [webantic/nginx-config-parser#10]
I'm just starting to test your parser but it does not seem to parse sites configs at all. Here's I'll just post example code with errors it produces for one of my domains config. Is that parser not build for domain configs at all or is it some error?

```
    let path = helpers.getNginxConfigProd(argv.client, argv.website)
    let parser = new ConfigParser()
    let config = parser.readConfigFile('/etc/nginx/sites-available/input.conf', { parseIncludes: false })

    console.log(parser.toConf(config))
    // server    [object Object];
    // server    [object Object];

    console.log(config)
    // {
    //     server: [
    //         {
    //             listen: [Array],
    //             server_name: 'domain.com www.domain.com',
    //             pagespeed: 'off',
    //             'location ~ ^/\\.well-known/acme-challenge': [Object],
    //             'location /': [Object]
    //         },
    //         {
    //             listen: [Array],
    //             server_name: 'domain.de www.domain.de',
    //             'location ~ ^/\\.well-known/acme-challenge': [Object],
    //             'location /': [Object]
    //         },
    //         server: [ [Object], [Object], server: [Object] ]
    // ]
    // }

    parser.readConfigFile('/etc/nginx/sites-available/input.conf')
    // ReferenceError: Unable to resolve include statement: "include snippets/wordpress.conf;".
```

File (had to add `.txt` at the end to upload it): 
[input.conf.txt](https://github.com/webantic/nginx-config-parser/files/5495915/input.conf.txt)

**Comment by carlevans719** [comment]
I get different output for this part of your example:
```javascript

    console.log(parser.toConf(config))
    // server    [object Object];
    // server    [object Object];
```

Could you let me know which version of this library and which version of node.js you are using so I can try using the same ones?

I noticed a bug when looking at this issue; the 3rd, 4th and 5th `server` blocks are appearing as properties of the parent array instead of elements in the array (i.e. `config.server.server` and `config.server.server.server`). I pushed a fix for that bug in #11. It might be that this was the cause of your issue too so could you please test with version 1.2.1 of this package from npm?

**Comment by popiolek** [comment]
It was version `1.2.0` . I've updated to `1.3.0` and it seems to work now. Thanks!
